### PR TITLE
Worked on the navigation bar, linked it to the appropriate section an…

### PR DESCRIPTION
**Pull request by 404atriumcode**
…d added the footer element. Also, added some CSS styles.

### CODE REVIEW

File: Assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* Navigation Bar Styles */
+ul {
```
Comment (RossNE99) [inline Assets/css/style.css]: Please replace "ul" with "nav ul" just so it doesnt effect all UL's on the page

File: Assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  width: 200px;
+  background-color: #f1f1f1;
+}
+
+li a {
```
Comment (RossNE99) [inline Assets/css/style.css]: Please replace "li a" with "nav ul li a" just so it doesnt effect all LI's on the page

File: Assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  display: block;
+  color: #000;
+  padding: 8px 16px;
+  text-decoration: none;
+}
+
+/* Change the link color on hover */
+li a:hover {
```
Comment (RossNE99) [inline Assets/css/style.css]: Please replace "li a:hover" with "nav ul li a:hover" just so it doesnt effect all LI's on the page

**Review by RossNE99 — CHANGES_REQUESTED** [review]
Please see comments

File: Assets/css/style.css (hunk 1 above)
Comment (404atriumcode) [reply Assets/css/style.css]: okay

**Review by RossNE99 — APPROVED** [review]

**Pull request merged**